.github/workflows/test.yml

```diff
@@ -110,7 +113,7 @@ jobs:
           DATABASE_ENGINE: django.db.backends.postgresql
           DATABASE_HOST: localhost
           DATABASE_USER: postgres
-          DATABASE_PASS: postgres
+          DATABASE_PASSWORD: postgres
```

Comment:
We've had this incorrect all this time 😦 

It's been working because then Django wouldn't pass any password to Postgres, and we didn't set `POSTGRES_PASSWORD` (now required in PostgreSQL 11).